src/platform/plugins/private/presentation_panel/public/panel_component/presentation_panel_error_internal.tsx

```diff
@@ -47,6 +60,19 @@ export const PresentationPanelErrorInternal = ({ api, error }: PresentationPanel
   }, [error]);
 
   const [label, setLabel] = useState('');
+  const [panelSize, setPanelSize] = useState<{ width: number; height: number } | undefined>(
+    undefined
+  );
+
+  useEffect(() => {
+    const observer = new ResizeObserver(() => {
+      if (!panelRef?.current) return;
+      const { width, height } = panelRef?.current?.getBoundingClientRect();
+      setPanelSize({ width, height });
+    });
+    if (panelRef?.current) observer.observe(panelRef?.current!);
+  }, [panelRef]);
```

Comment:
This allows the error state to be responsive to the panel being resized


![Dec-10-2025 13-26-15](https://github.com/user-attachments/assets/99416b8c-6d13-4887-ad14-a30d1cdc2db7)